### PR TITLE
Experience through Red Hat Event

### PR DESCRIPTION
**Pull request by jatinluthra14**

### CODE REVIEW

File: _posts/2018-01-11-jatin-jboss-event.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: "Transitioning from traditional to cloud-native development."
+description: "My expeience for attending RedHat webinar."
+headline:
+modified: 2018-01-11 21:20:35 +0530
+category: personal
+tags: []
+feature-img:
+mathjax:
+chart:
+author: 'Sidhya Tikku'
```
Comment (m-murad) [inline _posts/2018-01-11-jatin-jboss-event.md]: If you are copying someone else's work, please do it properly.

Comment (sidhyatikku) [reply _posts/2018-01-11-jatin-jboss-event.md]: Please @m-murad help me on this copyright infringement :'/

**Comment by sidhyatikku** [comment]
Copyright infringement notice 👎

**Comment by sidhyatikku** [comment]
Thanks for the free blog Jatin. I appreciate your philanthropy :)

**Comment by jatinluthra14** [comment]
@sidhyatikku @m-murad Our events are different. Just wanted the layout. I think I can copy the post layout.  I have not copied work. Sorry for inconvenience

**Comment by m-murad** [comment]
I know you wanted to copy the layout (I'm not suggesting that you cheated) but you should check your PR thoroughly before send it. It was sarcasm.

**Comment by jatinluthra14** [comment]
Sorry for inconvenience @sidhyatikku @m-murad. Thanks for trust

**Comment by m-murad** [comment]
No inconvenience, just fix the name.

**Comment by jatinluthra14** [comment]
Done

**Pull request merged**